src/game/puppyprint.c

```diff
@@ -599,6 +602,332 @@ void puppyprint_level_select_menu(void) {
     }
 }
 
+#ifdef BETTER_REVERB
+
+enum BetterReverbDebugFields {
+    PPREVERB_USE_LIGHTWEIGHT,
+    PPREVERB_DOWNSAMPLE_RATE,
+    PPREVERB_IS_MONO,
+    PPREVERB_FILTER_COUNT,
+    PPREVERB_WINDOW_SIZE,
+    PPREVERB_GAIN,
+    PPREVERB_GAIN_INDEX,
+    PPREVERB_REVERB_INDEX,
+    PPREVERB_DELAYS_L,
+    PPREVERB_DELAYS_R,
+    PPREVERB_REVERB_MULTS_L,
+    PPREVERB_REVERB_MULTS_R,
+    PPREVERB_AREA_ECHO,
+    PPREVERB_TOTAL,
+};
```

Comment:
this is puppyprint.c, i think it belongs in either puppyprint_debug or its own file

Reply:
well i made up a file that doesnt exist lol maybe make this the first `puppyprint_debug/page_reverb.c.in` file lmao

Reply:
Out of scope of this PR, but I agree this should probably happen for 2.1 even